Makefile.toml

```diff
@@ -79,4 +79,4 @@ args = [
 ]
 
 [tasks.serve]
-run_task = { name = ["build-highs", "dist-static-watch", "build-wasm-watch", "devserver"], fork = true, parallel = true }
+run_task = { name = ["dist-static-watch", "build-wasm-watch", "devserver"], fork = true, parallel = true }
```

Comment:
I'm curious about this change. Is the highs build broken?

Reply:
Nah, I just disabled it because I'm not using highs and it is taking my VM's disk space. Will add it back before merging.

Reply:
Cool, makes sense.